ds_judgements_public_ui/templates/pages/how_to_use_this_service.html

```diff
@@ -0,0 +1,199 @@
+      citation, court, party, judge and within a date range. These options can be combined on the <a
+              href="/search">More search options</a> page.</p>
+
+    <p>The latest published judgments and decisions are listed on the <a href="/">Find case law</a> page.
+      Alternatively, you can browse
+      by court and filter results.</p>
+
+    <h3>Search the text of every judgment and decision</h3>
+    <p>The search box on the <a href="/">Find case law</a> page lets you search using any search term including
+      a neutral citation, name(s) and keyword(s). This will search the full text of every judgment or decision we
+      have.</p>
+    <p>You do not need to do a Boolean search &ndash; spaces in between words will act as an 'and'. Results where the
+      words are closest together will appear as the most relevant at the top of the search results.</p>
+
+    <h3>Search specific fields</h3>
+    <p>Alternatively, you can search using one or more specific search terms on the <a href="/search">More search
+      options</a> page. You can use multiple search fields at the same time to narrow down your results. The
+      search boxes search particular fields. With the exception of keyword, these boxes do not search the full text of
+      judgments and decisions.</p>
+
+    <h4>Neutral citation</h4>
+    <p>The courts allocate a 'neutral citation' to judgments and decisions. This is a unique number and reference. The
+      neutral citation appears at the top of the judgment or decision, and is the official way to cite a judgment or
+      decision.</p>
+    <p>Examples of neutral citations include:</p>
+
+    <ul>
+      <li>[2021] EWCA Civ 1847</li>
+      <li>[2009] EWHC 2500 (Mercantile)</li>
+      <li>[2021] EWCOP 18 [2019] UKSC 38</li>
+    </ul>
+
+    <p>They typically include:</p>
+
+    <ul>
+      <li>The year the judgment or decision was handed down. For example: [2009]</li>
+      <li>The standard court acronym. For example: EWCA is England and Wales Court of Appeal</li>
+      <li>The standard acronym for the division of court. For example: Civ is Civil Division</li>
+      <li>A unique number allocated by the individual court within a given year. For example: '1847' will be the
+        unique number for the Court of Appeal Civil Division in 2021
+      </li>
+    </ul>
+
+    <p>If you are struggling to find a judgment or decision using a neutral citation, first check:</p>
+
+    <ul>
+      <li>You have written the neutral citation in full</li>
+      <li>There are no typos or mistakes</li>
+      <li>The spaces are in the appropriate places</li>
+      <li>This is the neutral citation allocated by the court rather than the case number or a reference number issued
+        by a publisher
+      </li>
+    </ul>
+
+    <p>You can also use neutral citations allocated by the British and Irish Legal Information Institute (BAILII) for
```

Comment:
Might be worth adding that you'll need to drop the `B` from the numbering of such citations.